### ISSUE TITLE
1.1.15 release broken missing instance_types.json?

### ISSUE THREAD

**Issue by tardyp** [spulec/moto#1218]
Please see error in 
https://nine.buildbot.net/#/builders/8/builds/1378

```
    from .models import acm_backends
  File "/buildbot/buildbot-job/build/sandbox/local/lib/python2.7/site-packages/moto/acm/models.py", line 7, in <module>
    from moto.ec2 import ec2_backends
  File "/buildbot/buildbot-job/build/sandbox/local/lib/python2.7/site-packages/moto/ec2/__init__.py", line 2, in <module>
    from .models import ec2_backends
  File "/buildbot/buildbot-job/build/sandbox/local/lib/python2.7/site-packages/moto/ec2/models.py", line 115, in <module>
    INSTANCE_TYPES = json.load(open(os.path.join(RESOURCES_DIR, 'instance_types.json'), 'r'))
exceptions.IOError: [Errno 2] No such file or directory: u'/buildbot/buildbot-job/build/sandbox/local/lib/python2.7/site-packages/moto/ec2/resources/instance_types.json'
```
I didn't see that error with 1.1.13.

Is this a problem with the release scripts?

**Comment by JackDanger** [comment]
It was an oversight in bundling json files in `MANIFEST.in`. It's fixed in the current release!

**Comment by terrycain** [comment]
My bad should of caught that too.

**Comment by terrycain** [comment]
@JackDanger This should be fixed right seeing as were on 1.1.22

**Comment by JackDanger** [comment]
Yup, this is all fixed.
